plugins/com.cubrid.cubridmigration.command/src/com/cubrid/cubridmigration/command/TableStatus.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation.
+ * Copyright (C) 2016 CUBRID Corporation.
+ *

```

Comment:
```suggestion
/*
 * Copyright (C) 2016 CUBRID Corporation.
 *
```
`Copyright (C) 2016 CUBRID Corporation.` 만 사용해도 됩니다.

Reply:
수정하겠습니다.